BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/model/Status.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.pontem.model;
+
+/** Represents the Status of executed Queries and Workloads */
+public enum Status {
```

Comment:
can we be more detailed here with the name of Status (e.g., WorkloadJobStatus) OR can we not be more detailed wwith the naming as we are re-using Status for multiple things?

Reply:
We are using the status for both queries and workloads so that's why I left it generic